components/samsung_ac/protocol_non_nasa.cpp

```diff
@@ -538,7 +538,7 @@ namespace esphome
 
         void process_non_nasa_packet(MessageTarget *target)
         {
-            if (debug_log_packets)
+            if (debug_log_messages)

```

Comment:
This did not do what you would like to archive. This also print the undefined messages.